### DISCUSSION TITLE
#️⃣ Meeting Notes: Cometa X ASA Stats discuss news from each project

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#297] [Meeting]
@AlgoRhythMatic  and @ipaleka met with Geo from Cometa

* Discussed Cometa UI and provided feedback
* Discussed ASA Stats news
    * Mobile Apps
    * Governance
    * Utility tokens
    * API + widgets
    * Liquidity (Tinyman) + internal rewards program
* Discussed Cometa LAAS (Liquidity as a service)
    * Uses a Vault
    * No rewards needed - projects simply provide their side of the pair (ASASTATS in our case).
    * All trading fees are paid back to users as APR
    * Investors only need provide the other side of the pairing - ALGO, USDC, etc - which reduces impermanent loss
* Discussed “zap” functionality, and potential for it to be used via API in widget in the future
* Geo provided a followup LAAS video via Telegram (attached):
https://user-images.githubusercontent.com/97758273/197106124-a3226ccd-deda-44bf-b6a0-332ac69e2e2e.mp4

ToDo: present video to the community + understand interest and/or followup questions for Geo and the Cometa team.

**Comment by ipaleka** [comment]
Great potential with this, impermanent loss compensation could be a game changer.
